src/style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--0 div:nth-child(n) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 div:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 div:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 div:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 div:nth-child(n+3) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--3 div:nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 div:nth-child(n+4) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--4 div:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 div:last-child {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 div:nth-child(n) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
it can be shorter use something like that `.stars--3 .stars__star:nth-child(-n + 3)` and combine this selectors